src/components/DownloadButton.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <DropdownButton v-if="filesizes">
+    <template #default="{ buttonProps }">
+      <a
+        v-bind="buttonProps"
+        class="whitespace-nowrap"
+        :href="selectedFormat.download_url"
+        download=""
+      >
+        <span>{{ $t('download-button.download') }}</span>
+        <span class="ml-4 selected-format">
+          {{ selectedFormat.extension_name }}
+        </span>
+        <span class="ml-1 font-thin">{{
+          getFormatSize(selectedFormat.extension_name)
+        }}</span>
+      </a>
+    </template>
+
+    <template
+      #items="{
+        activeItemClass,
+        itemClass,
+        itemA11yProps,
+        toggleOpen,
+        onItemKeydown,
+      }"
+    >
+      <ul>
+        <li v-for="format in formats" :key="format.name">
+          <button
+            class="flex justify-between w-full"
+            :class="[
+              itemClass,
+              selectedFormat.extension_name === format.extension_name
+                ? activeItemClass
+                : '',
+            ]"
+            type="button"
+            v-bind="itemA11yProps"
```

Comment:
Can we change the `aria-label` from dropdown to something like 'Select the download file format'?